src/lib/deploy-helpers.ts

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2023-Present The Pepr Authors
+
+import { K8s, kind, KubernetesListObject } from "kubernetes-fluent-client";
+import Log from "./logger";
+
+// get all deployments
+export async function getDeployments() {
+  const deployments = await K8s(kind.Deployment).InNamespace("pepr-system").Get();
+  return deployments;
+}
+
+// waits for all replicas to be ready for all deployments
```

Comment:
update comment for deployment in pepr-system namespace

Reply:
update function name too. Not sure that this needs to be a function

Reply:
Updated comment and consolidated functions into one function